api/v1alpha2/types.go

```diff
@@ -169,6 +169,10 @@ type NetworkSpec struct {
 	// Subnets configuration.
 	// +optional
 	Subnets Subnets `json:"subnets,omitempty"`
+
+	// ControlPlaneLoadBalancerScheme (defaults to Internet-facing)
+	// +optional
+	ControlPlaneLoadBalancerScheme *ClassicELBScheme `json:"controlPlaneLoadBalancerScheme,omitempty""`
```

Comment:
It'd be great if we can have a struct encapsulating all the knobs we want to expose specifically for the API Server load balancer and have this optional field (and other later probably) in there

Reply:
Thanks! I'd be happy for this PR to be closed and I'll provide a separate PR with a proposed implementation.

Reply:
No probs.

/close